### ISSUE TITLE
Shouldn't resilience4j's time limiter respect Feign timeouts?

### ISSUE THREAD

**Issue by eduardo-bueno-ifood** [spring-cloud/spring-cloud-circuitbreaker#90]
Working with OpenFeign, Resilience4j completely disregards any connectTimeout/readTimeout configured in a client. I have tested with specific clients as well as `feign.client.config.default`. It would be nice to have things working in tandem.

Additionally, Resilience4j's documentation is really confusing and I can't find an equivalent of this Hystrix configuration:
```
hystrix:
  command:
    default:
      execution:
        isolation:
          thread:
            timeoutInMilliseconds: 5000
```
I tried it like below but it didn't work. Would appreciate if anyone knows how to tweak the default timeout.
```
resilience4j.timelimiter:
  instances:
    backendA:
      timeoutDuration: 5s
    backendB:
      timeoutDuration: 5s
```

**Comment by ryanjbaxter** [comment]
There is also a problem with OpenFeign and Hystrix
https://docs.spring.io/spring-cloud-openfeign/docs/2.2.7.RELEASE/reference/html/#timeout-handling

Whatever timeout elapses first is going to "win".

Currently properties configuration is not supported #61, there is a PR to support it.

To configure timeouts read this documentation
https://docs.spring.io/spring-cloud-circuitbreaker/docs/current/reference/html/#default-configuration

**Comment by spring-cloud-issues** [comment]
If you would like us to look at this issue, please provide the requested information. If the information is not provided within the next 7 days this issue will be closed.

**Comment by eduardo-bueno-ifood** [comment]
Thank you. That fills the gap, yeah. It would be really nice to have to set a client timeout only once but one can imagine how difficult this should be. It also might not make sense in other cases.
